File: JavaNgArch/src/projectbase/bd/BizException.java
package projectbase.bd;

import projectbase.utils.IErrorForUser;
    public class BizException extends Exception implements IErrorForUser
    {
        /**
		 * 
		 */
		private static final long serialVersionUID = 1L;
		public BizException(){}
        public BizException(String msg)  { super(msg);}

    }

File: NetNgArch/src/ProjectBase/BusinessDelegate/BizException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ProjectBase.Utils;

namespace ProjectBase.BusinessDelegate
{
    public class BizException :Exception, IErrorForUser
    {
        public BizException() : base(){}
        public BizException(string msg) : base(msg) { }

    }
}
